pico_sdk_import.cmake

```diff
@@ -5,17 +5,17 @@
 # It should be include()ed prior to project()
 
 if (DEFINED ENV{PICO_SDK_PATH} AND (NOT PICO_SDK_PATH))
-    set(PICO_SDK_PATH $ENV{PICO_SDK_PATH})
+    set(PICO_SDK_PATH "$ENV{PICO_SDK_PATH}")
     message("Using PICO_SDK_PATH from environment ('${PICO_SDK_PATH}')")
 endif ()
 
 if (DEFINED ENV{PICO_SDK_FETCH_FROM_GIT} AND (NOT PICO_SDK_FETCH_FROM_GIT))
-    set(PICO_SDK_FETCH_FROM_GIT $ENV{PICO_SDK_FETCH_FROM_GIT})
+    set(PICO_SDK_FETCH_FROM_GIT "$ENV{PICO_SDK_FETCH_FROM_GIT}")
```

Comment:
`PICO_SDK_FETCH_FROM_GIT` is a BOOL value, so will never contain a space (AFAIK)

Reply:
As above, and as far as I can think, no, a boolean won't ever contain a space.